sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Triggers/EventHubTriggerAttributeBindingProvider.cs

```diff
@@ -54,9 +54,15 @@ public Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext contex
             Func<ListenerFactoryContext, bool, Task<IListener>> createListener =
              (factoryContext, singleDispatch) =>
              {
+                 var options = _options.Value;
+                 if (singleDispatch && !options.IsSingleDispatchEnabled)
```

Comment:
Why do this? We already have a test for single dispatch so this code will be hit.

Reply:
or is it still possible to bind to this via config even though it is now internal?